lib/location/method_gnss.c

```diff
@@ -1175,38 +1142,18 @@ static void method_gnss_agnss_expiry_process(const struct nrf_modem_gnss_agnss_e
 	}
 }
 
-/* Queries assistance data need from GNSS.
- *
- * To maintain backward compatibility with older modem firmware versions, two different methods are
- * supported. If the nrf_modem_gnss_agnss_expiry_get() API is not supported (pre-v1.3.2 modem
```

Comment:
I cannot add comment to old line 1258 but there are similar comments about if this is not supported etc.